### DISCUSSION TITLE
Provision of date/time range to narrow down on particular timeframe when the issue is observed

### DISCUSSION THREAD

**Discussion by shaqb** [ludeknovy/jtl-reporter#218] [Ideas]
Hi @ludeknovy - really liking the tool. We've been using it heavily during our long soak tests. One thing we would like (unless it's there and I can't see it), is the ability to set a global date/time for a particular test run so that when an issue occurs we can zoom into that period.

For example, we have a situation where we were seeing spikes in response times during a 48h soak test, and it would be good to filter on the time period when the issue occured. Otherwise we lose granularity when we look at the test as a whole.

It would also be good if when we zoom in on a graph we could apply that same zoom to all graphs?

**Comment by ludeknovy** [comment]
Hi @shaqb ! I like the idea and I will have a look at it.
